### ISSUE TITLE
[tasks/gulp.watch] - Should not watch changes in bower_components

### ISSUE THREAD

**Issue by ericmdantas** [ericmdantas/generator-ng-fullstack#279]
As reported by @fernandacanto, when using `ng1` + `bower` and using a module such as `angular-i18n`, the app page is never fully loaded. 

That happens because the watch task is trying to map/watch everything inside `bower_components`, and that can be very slow/heavy, because some of the folders have way too many `.js` files - which is the case with `angular-i18n`.

It turns out this (in tasks/watch):

```js
  _watchable.push(base.DEV + "**/*.js");
  _watchable.push(base.DEV + "**/*.css");
  _watchable.push(base.DEV + "**/*.html");
```

should be this:

```js
  _watchable.push(`${base.DEV}**/*.js`);
  _watchable.push(`${base.DEV}**/*.css`);
  _watchable.push(`${base.DEV}**/*.html`);
  _watchable.push(`!${base.DEV}bower_components/**/*`);
```

**Comment by ericmdantas** [comment]
Solved by 255ee74.

**Comment by ericmdantas** [comment]
Released `1.9.18` with the fix.

**Comment by ericmdantas** [comment]
Also released in the `next` branch.

```shell

$ npm i -g generator-ng-fullstack # to download from '1.9' branch

``` 

```shell

$ npm i -g generator-ng-fullstack@next # to download from 'next' branch

```
